tests/pubgrade/endpoints/test_builds.py

```diff
@@ -132,152 +148,174 @@ class TestBuild:
     repository_url = "https://github.com/elixir-cloud-aai/drs-filer"
 
     def setup(self):
-        self.app.config['FOCA'] = \
-            Config(db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG)
-        self.app.config['FOCA'].db.dbs['pubgradeStore']. \
-            collections['repositories'].client = mongomock.MongoClient(
-        ).db.collection
+        self.app.config["FOCA"] = Config(
+            db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG
+        )
+        self.app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+            "repositories"
+        ].client = mongomock.MongoClient().db.collection
         for repository in MOCK_REPOSITORIES:
-            self.app.config['FOCA'].db.dbs['pubgradeStore']. \
-                collections['repositories'].client.insert_one(
-                repository).inserted_id
-        self.app.config['FOCA'].db.dbs['pubgradeStore']. \
-            collections['builds'].client = mongomock.MongoClient(
-        ).db.collection
+            self.app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+                "repositories"
+            ].client.insert_one(repository)
+        self.app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+            "builds"
+        ].client = mongomock.MongoClient().db.collection
 
     def setup_with_build(self):
         self.setup()
-        self.app.config['FOCA'].db.dbs['pubgradeStore']. \
-            collections['builds'].client.insert_one(
-            MOCK_BUILD_INFO).inserted_id
-        self.app.config['FOCA'].db.dbs['pubgradeStore']. \
-            collections['builds'].client.insert_one(
-            MOCK_BUILD_INFO_2).inserted_id
-
-    @patch('pubgrade.pubgrade.endpoints.builds.create_build',
-           mocked_create_build)
+        self.app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+            "builds"
+        ].client.insert_one(MOCK_BUILD_INFO)
+        self.app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+            "builds"
+        ].client.insert_one(MOCK_BUILD_INFO_2)
+
+    @patch(
+        "pubgrade.modules.endpoints.builds.create_build", mocked_create_build
+    )
     def test_register_builds(self):
         self.setup()
         with self.app.app_context():
-            res = register_builds(MOCK_REPOSITORIES[1]['id'],
-                                  MOCK_REPOSITORIES[1]['access_token'],
-                                  MOCK_BUILD_PAYLOAD)
+            res = register_builds(
+                MOCK_REPOSITORIES[1]["id"],
+                MOCK_REPOSITORIES[1]["access_token"],
+                MOCK_BUILD_PAYLOAD,
+            )
             assert isinstance(res, dict)
-            assert 'id' in res and res['id'][:6] == MOCK_REPOSITORIES[1]['id']
+            assert "id" in res and res["id"][:6] == MOCK_REPOSITORIES[1]["id"]
```

Comment:
This kind of "magic numbers" need to be documented. Why 6 and not 42? We need an explanation to be written in the code.